### ISSUE TITLE
Test Issue -Dashboard workflow

### ISSUE THREAD

**Issue by prijo19** [rokwire/illinois-app#1627]
***Describe the new panel (a single visualization) or dashboard (a collection of visualizations)***
A clear and concise description of what you'd like to see.
If possible, please draw a mockup of the dashboard or report layout, take a picture, and upload the image. 

This is a test message for the workflow
 
 
***What are you hoping to measure?***
A description of the metrics that should be used (e.g., unique users, sessions, etc.).
 This is a test message for the workflow

 
***What is the timeframe that should be used?***
A description of the timeframe used to filter metrics (e.g., the last 24 hours, month-to-date, etc.).
 This is a test message for the workflow

***Who should have access to the dashboard?***
A description of the target audience that should be granted access to the dashboard. 
 This is a test message for the workflow
 
***By when should the request be fulfilled?***
This is a test message for the workflow

**Comment by julianchin** [comment]
Thank you Prijo. This is to confirm we receive the test issue.

**Comment by ahetrick** [comment]
Thanks @ToddNelson! Does it make sense to have Bill as an initial assignee (along with Julian and myself), as his approval is needed for the first phase? That way, he can easily see the template and just reply via email if the request is approved. What do you think?

**Comment by ToddNelson** [comment]
Let’s bring this up during the meeting on the 19th…. My opinion is not including him as the original assignee, with both you and Ashley manually kicking off the approval process as needed. Reason being, I don’t think Bill should bear the responsibility of kicking off the process. Is there a first step the metrics team should take (like clarifying the request if needed) before approval is requested?

**Comment by ahetrick** [comment]
@ToddNelson Certainly, one version of the process could begin with the analytics team replying with any clarifying questions, and then adding Bill to approve or deny when we're ready. I'm drafting a document that has two versions of the process currently (we're open to more, of course). Would you be able to take a look after the analytics team has reviewed, and let us know your thoughts?
